source/release/4.2.0/index.html.md

```diff
@@ -0,0 +1,710 @@
+
+- An all new **VM Portal** for non-admin users - designed with React-based UI and Patternfly principles - replaces the existing User Portal. Built with performance and ease of use in mind, the VM Portal keeps the Extended view of the User Portal, plus a streamlined functionality. 
+
+- A new **High Performance** virtual machine (VM) type has been added to the New VM dialog box in the Administration Portal. By selecting the ‘High Performance’ option in the ‘Optimized for’ field, administrators can effortlessly optimize a VM for high performance workloads.
+
+- **Open Virtual Network (OVN)** adds support for Open vSwitch virtual networking. oVirt VMs can now use logical overlay networks defined by OVN, allowing the user to manage and define multiple logical networks via one physical network. OVN is managed either via the oVirt Administration Portal, or REST.  For more information, see the OVN feature page. 
+
+- oVirt now supports **Nvidia vGPU**, a technology that enables users to shard a GRID capable physical GPU - such as Nvidia Tesla M60 - into number of smaller instances. Each instance can be assigned to a VM, for GPU-accelerated workloads.
+ 
+- The **ovirt-ansible-roles** package helps users with common administration tasks. All roles can be executed from the command line using Ansible, and some are executed directly from oVirt engine. You can learn [how to use oVirt Ansible roles](https://ovirt.org/blog/2017/08/ovirt-ansible-roles-how-to-use/) on the oVIrt blog.
+
+- **Virt-v2v** - the tool that converts VMs from a foreign hypervisor to run on KVM - now supports **Debian/Ubuntu** based VMs, in addition to the supported RPM and Windows-based VMs. It is available for VDSM hosts running on RHEL 7.4, and from oVirt hosts versions 4.0 and above.
+
+- oVirt 4.2.0 uses **PostgresSQL 9.5** as its database, for improved performance.
+
+- Support is now restored for **Gluster ISO domains**, without the need for NFS which was previously disabled by Gluster. 
+
+- **Affinity labels** create a positive affinity between the hosts and VMs to which they are applied. The new Affinity Labels sub tab for clusters, hosts, and VMs in the Administration Portal provides a table view of labels associated with the currently selected entity, as well as the option to add, edit, and delete them. Additionally, existing labels can be added and/or removed from VMs and hosts in their respective new/edit popup dialogs.
+
+To learn about features introduced before 4.2.0, see the [release notes for previous versions](/documentation/#previous-release-notes).
+
+
+## Install / Upgrade from previous versions
```

Comment:
I am not clear on the structure / relationship between these three headings:
## Install / Upgrade from previous versions
### Fedora / CentOS / RHEL
## ALPHA RELEASE